lib/pbench/server/database/models/users.py

```diff
@@ -0,0 +1,132 @@
+from flask_bcrypt import generate_password_hash
+from email_validator import validate_email
+from sqlalchemy import Column, Integer, String, DateTime, LargeBinary
+from pbench.server.database.database import Database
+from sqlalchemy.orm import relationship
+from sqlalchemy.orm import validates
```

Comment:
Or `from sqlalchemy.orm import relationship, validates`, to keep them connected?

Reply:
yes